### DISCUSSION TITLE
Merging 10x Multiome

### DISCUSSION THREAD

**Discussion by rhart604** [stuart-lab/signac#706] [Q&A] [answered]
Hi,

I'm following the [10x RNA/ATAC vignette](https://satijalab.org/signac/articles/pbmc_multiomic.html) and everything works well with one sample, but when I try to merge two samples (without integration) I'm having trouble.  Do you have an example script or vignette that shows best practices for merging two samples?  And what's the best stage to FindClusters?  Is this done with only the RNA assay?

Thanks!

**Comment by timoast** [comment, marked as answer]
See https://satijalab.org/signac/articles/merging.html

**Reply by rhart604** [reply]
OK, but does this get run on the atac_fragments.tsv.gz file or do I first have to extract the ATAC results from the filtered_feature_bc_matrix.h5 file?

**Reply by timoast** [reply]
You need to run the `merge` function on your Seurat objects (explained in the vignette linked above)

**Reply by rhart604** [reply]
My atac_singlecell.csv file does not have a passed_filters field.  Only is_cell_barcode and excluded_reason fields are found.  I used cellranger-arc v. 2.0.0.  Is it appropriate to filter instead on excluded_reason == 0?

**Reply by timoast** [reply]
For new questions please open a new discussion topic
